common/app/routes/challenges/views/backend/Back-End.jsx

```diff
@@ -117,13 +118,16 @@ export class BackEnd extends PureComponent {
     return (
       <Row>
         <Col
-          md={ 6 }
-          mdOffset={ 3 }
-          xs={ 12 }
+          xs={ 6 }
```

Comment:
@raisedadead I had the wrong value here, `12` rather than `6`, which was causing that behavior. I must have accidentally reverted it before making the last PR.